src/vips/voperation.lua

```diff
@@ -167,58 +163,56 @@ local voperation_mt = {
             for i = 1, #arguments do
                 local flags = arguments[i].flags
 
-                if band(flags, INPUT) ~= 0 and 
-                    band(flags, REQUIRED) ~= 0 and 
-                    band(flags, DEPRECATED) == 0 then
+                if band(flags, INPUT) ~= 0 and
+                        band(flags, REQUIRED) ~= 0 and
+                        band(flags, DEPRECATED) == 0 then
                     n_required = n_required + 1
                 end
             end
 
             -- so we should have been passed n_required, or n_required + 1 if
             -- there's a table of options at the end
             local last_arg
-            if #call_args == n_required then 
+            if #call_args == n_required then
                 last_arg = nil
             elseif #call_args == n_required + 1 then
                 last_arg = call_args[#call_args]
                 if type(last_arg) ~= "table" then
                     error("unable to call " .. name .. ": " .. #call_args ..
-                        " arguments given, " .. n_required .. 
-                        ", but final argument is not a table")
+                            " arguments given, " .. n_required ..
+                            ", but final argument is not a table")
                 end
             else
                 error("unable to call " .. name .. ": " .. #call_args ..
-                    " arguments given, but " .. n_required ..  " required")
+                        " arguments given, but " .. n_required .. " required")
             end
 
             -- the first image argument is the thing we expand constants to
             -- match ... look inside tables for images, since we may be passing
             -- an array of image as a single param
-            local match_image = find_order(
-                function(x)
-                    if Image.is_Image(x) then
-                        return x
-                    else
-                        return nil
-                    end
-                end,
-                call_args
-            )
+            local match_image = find_order(function(x)
+                if Image.is_Image(x) then
+                    return x
+                else
+                    return nil
+                end
+            end,
+                call_args)
```

Comment:
This indenting looks very odd! But I guess if it's what the linter likes. 

Perhaps we should swap the args to find_order()?

Reply:
Hopefully fixed with: https://github.com/kleisauke/lua-vips/commit/b8917b960ad44f1d325d946caedbd47f44ca94c7. The auto-reformat command from my IDE does sometimes strange things..